.github/workflows/lint_test_build.yml

```diff
@@ -42,8 +42,8 @@ jobs:
     needs: checkout
     timeout-minutes: 5
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
I'm not 100% sure, but are these two lines even necessary if we are doing:
`uses: ./.github/workflows/rafiki/env-setup` where the workflow already does 

```yaml
    - uses: actions/checkout@v3
    - uses: actions/setup-node@v3
```
?

Reply:
It seems like they can be removed throughout this file wherever `env-setup` is used